otter/test/test_cloud_client.py

```diff
@@ -414,36 +389,50 @@ def test_post_and_delete_not_the_same(self):
             {"cloud_client": {"throttling": {"create_server_delay": 1,
                                              "delete_server_delay": 0.4}}})
         clock = Clock()
-        deleter = _default_throttler(clock, ServiceType.CLOUD_SERVERS,
-                                     'delete')
-        poster = _default_throttler(clock, ServiceType.CLOUD_SERVERS, 'post')
+        locks = WeakLocks()
+        deleter = _default_throttler(
+            locks, clock, ServiceType.CLOUD_SERVERS, 'delete')
+        poster = _default_throttler(
+            locks, clock, ServiceType.CLOUD_SERVERS, 'post')
         self.assertIsNot(deleter, poster)
 
-    def test_post_delay_configurable(self):
-        """The delay for creating servers is configurable."""
+    def _test_throttle(self, cfg_name, stype, method):
+        """Test a specific throttling configuration."""
+        locks = WeakLocks()
         set_config_data(
-            {'cloud_client': {'throttling': {'create_server_delay': 500}}})
+            {'cloud_client': {'throttling': {cfg_name: 500}}})
         self.addCleanup(set_config_data, {})
         clock = Clock()
-        bracket = _default_throttler(clock, ServiceType.CLOUD_SERVERS, 'post')
+        bracket = _default_throttler(locks, clock, stype, method)
+        if bracket is None:
+            self.fail("No throttler for %s and %s" % (stype, method))
```

Comment:
Nitpick. You could instead do `self.assertIsNotNone(bracket, "No throttler for {} and {}".format(stype, method))`